src/Appium.Net/Appium/ImageComparison/ComparisonResult.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+
+namespace OpenQA.Selenium.Appium.ImageComparison
+{
+    public abstract class ComparisonResult
+    {
+        /// <summary>
+        /// The visualization of the matching result represented as base64-encoded PNG image.
+        /// </summary>
+        public string Visualization
+        {
+            get { return Result["visualization"].ToString(); }
```

Comment:
@mykola-mokhnach 
concerning "wrapper type for png bitmaps"
since the java-client also simply returns the base64 string, I think it is appropriate here as well 
(this way it is consistent w/ the parameters for the api-methods and provides more flexibility)

Reply:
I'm fine with that